pkgconfig/libcrypto.pc.in

```diff
@@ -4,6 +4,6 @@ includedir=@includedir_for_pc_file@
 
 Name: AWS-LC-libcrypto
 Description: AWS-LC cryptography library
-Version: @VERSION@
+Version: 1.1.1
```

Comment:
Could we still abstract this into `CMakeLists` to make the pkgconfig files easier to change?